### PR TITLE
need your review for this

### PR DESCRIPTION
**Pull request by ghost**
so, here it is, the crossplatform Warfork, if you decide to merge this PR, I assume that you will only:
* officially provide only 32 bit Warfork for both 32 bit and 64 bit Windows
* officially provide only 64 bit Warfork for both Linux and macOS
* let other people build binaries they need themselves (64 bit Warfork on Windows, 32 bit Warfork on Linux, both 32 bit and 64 bit Warfork on FreeBSD, NetBSD, OpenBSD and illumos, 64 bit Warfork on DragonFly BSD)

### CODE REVIEW

**Comment by goleo108** [comment]
reminder: binaries and libraries are no longer suffixed, you are required to not only copy warfork, wf_server and wftv_server, but also the contents of basewf and libs directories

**Comment by goleo108** [comment]
reminder x2: while modules_21.pk3 is still present, temporarily disabling pure checks is required to connect to servers unless you host a server without modules_21.pk3

File: third-party/libRocket/Source/Core/Dictionary.cpp [hunk 1]
```diff
@@ -479,10 +479,9 @@ void Dictionary::Clear()
 	DictionaryEntry *ep;
 	bool table_is_malloced;
 	int n_full;
-	int i, n;
+	int i;
```
Comment (Jan200101) [inline third-party/libRocket/Source/Core/Dictionary.cpp]: n is still used on Line 493 in an assert statement

Comment (goleo108) [reply third-party/libRocket/Source/Core/Dictionary.cpp]: I love that it passes build on Linux, it appears that ROCKET_ASSERT is defined to empty

**Comment by goleo108** [comment]
if you can fix issues with static linking, then fuck everything I said above, otherwise it's save to merge everything before static linking commits

**Comment by goleo108** [comment]
good luck to anyone who wants to fix runtime issues when statically linking game and cgame, I am out

**Comment by Warfork** [comment]
Hey there, I didn't see you in #ravenports on freenode. Stop by Discord again if you're feeling up to it and lets try to get this figured out.

**Comment by goleo108** [comment]
@Warfork I was disconnected from freenode because compiling llvm consumed all my RAM, also I don't want to waste time on useless talks about the same thing with your advisors, I figured everything I did was a waste a time, now I am busy getting this game to Ravenports because I made a promise, after I am done I don't care what happens to this game, I won't play it because I am irritated very much, good luck looking for new maintainer who can make this shit code elegant enough to make it build and run even on illumos

**Pull request closed without merging**